pokedex/src/data/LocationTypes.ts

```diff
@@ -0,0 +1,39 @@
+export interface Location {
+    lat: number;
+    lng: number;
+  }
+  
+export interface MapComponentProps {
+    center: Location;
+    pokemonLocation: Location;
+    isLoadedMap: boolean;
+    pokemonIcon: string;
+    moveoIcon: string;
+  }
+  
+ export  const containerStyle = {
+    width: "500px",
+    height: "500px",
+  };
+  
+ export const MoveoLocation = {
+    latitude: 32.064584,
+    longitude: 34.771829,
+  };
+  
+ export const apiKey = "AIzaSyCbRoE1LBwpa95FBvWtzkigLzRZl4YuMGQ"; 
```

Comment:
Please try and enter it to '.env' file.
after you enter it to '.env' file, try and run your project and see if it work properly, and only then push the update.
because event if there wasn't anything like "env" file, look at the name of the file ("LocationTypes"). 
Key is doesn't a type and doesn't belong here anyway